lib/generators/paper_trail/update_sti/USAGE

```diff
@@ -0,0 +1,2 @@
+Description:
+  Generates (but does not run) a migration to update item_type for STI entries in an existing versions table.
```

Comment:
Nice idea to have a USAGE file here, but please add link to full docs in readme.